fastlane/lib/fastlane/plugins/plugin_info_collector.rb

```diff
@@ -37,9 +37,11 @@ def collect_plugin_name(initial_name = nil)
 
         break if plugin_name_valid?(plugin_name)
 
-        if plugin_name_taken?(plugin_name)
-          # Plugin name is already taken on RubyGems
-          @ui.message("\nPlugin name '#{plugin_name}' is already taken on RubyGems, please choose a different one.")
+        gem_name = PluginManager::FASTLANE_PLUGIN_PREFIX + plugin_name
```

Comment:
At this point, might `plugin_name` contain the prefix already? What if the user enters `fastlane-plugin-yolo`?